atc/db/team.go

```diff
@@ -151,77 +151,7 @@ func (t *team) FindWorkerForVolume(handle string) (Worker, bool, error) {
 }
 
 func (t *team) Containers() ([]Container, error) {
-	rows, err := selectContainers("c").
-		Join("workers w ON c.worker_name = w.name").
-		Join("resource_config_check_sessions rccs ON rccs.id = c.resource_config_check_session_id").
-		Join("resources r ON r.resource_config_id = rccs.resource_config_id").
-		Join("pipelines p ON p.id = r.pipeline_id").
-		Where(sq.Eq{
-			"p.team_id": t.id,
-		}).
-		Where(sq.Or{
-			sq.Eq{
-				"w.team_id": t.id,
-			}, sq.Eq{
-				"w.team_id": nil,
-			},
-		}).
-		Distinct().
-		RunWith(t.conn).
-		Query()
-	if err != nil {
-		return nil, err
-	}
-
-	var containers []Container
-	containers, err = scanContainers(rows, t.conn, containers)
-	if err != nil {
-		return nil, err
-	}
-
-	rows, err = selectContainers("c").
-		Join("workers w ON c.worker_name = w.name").
-		Join("resource_config_check_sessions rccs ON rccs.id = c.resource_config_check_session_id").
-		Join("resource_types rt ON rt.resource_config_id = rccs.resource_config_id").
-		Join("pipelines p ON p.id = rt.pipeline_id").
-		Where(sq.Eq{
-			"p.team_id": t.id,
-		}).
-		Where(sq.Or{
-			sq.Eq{
-				"w.team_id": t.id,
-			}, sq.Eq{
-				"w.team_id": nil,
-			},
-		}).
-		Distinct().
-		RunWith(t.conn).
-		Query()
-	if err != nil {
-		return nil, err
-	}
-
-	containers, err = scanContainers(rows, t.conn, containers)
-	if err != nil {
-		return nil, err
-	}
-
-	rows, err = selectContainers("c").
-		Where(sq.Eq{
-			"c.team_id": t.id,
-		}).
-		RunWith(t.conn).
-		Query()
-	if err != nil {
-		return nil, err
-	}
-
-	containers, err = scanContainers(rows, t.conn, containers)
-	if err != nil {
-		return nil, err
-	}
-
-	return containers, nil
+	return getContainersByTeamIds([]int{t.id}, false, t.conn)
```

Comment:
Do we need to bother with skipping the team name here (the `false`)? What's the harm in just always including it?